src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2699,15 +2710,25 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         // Initialize the animation if it does not exist
         // We only initialize here instead of in the ctor because depending on the bell style setting,
         // we may never need this animation
-        if (!_bellLightAnimation)
+        if (!_bellLightAnimation && !_isBackgroundLight)
         {
             _bellLightAnimation = Window::Current().Compositor().CreateScalarKeyFrameAnimation();
             // Add key frames and a duration to our bell light animation
-            _bellLightAnimation.InsertKeyFrame(0.0, 2.0);
-            _bellLightAnimation.InsertKeyFrame(1.0, 1.0);
+            _bellLightAnimation.InsertKeyFrame(0.0, 4.0);

```

Comment:
oh, right, okay, I get it. I was all confused why the current animation was moved to the light-colored BGs. 

Before, we had a `White` light, that we animated from 2->1 `Intensity`. Now we've got a `Gray` light, so for dark BGs, we need a different animation. It's coincidental that the light BG version of the animation just so happens to use 1->2 `Intensity` (but now that I've typed it, I'm better realizing that's not even the same values as before 😅)